user_sync/connector/directory_okta.py

```diff
@@ -112,7 +112,7 @@ def load_users_and_groups(self, groups, extended_attributes):
         is_using_source_filter = False
         source_filters = options['source_filters']
         source_filter = source_filters.get('all_users_filter')
-        if (source_filter != None):
+        if source_filter is not None:
```

Comment:
You're right - I had to either commit or discard that change to switch branches (I hate dealing with stashes).  Equality operators should not be used in None comparisons, so I wanted to fix that.

I'm still making changes to the module.  We need to refactor user input workflow.